Ogutu-Brian/Questioner
# #163028251 Add continuous Integration to the project

### What does this PR do?
It adds Continuous Integration to the project
### Description of Task to be completed?
Have the following achieved
1. Integrate TravisCI to github repository
2. Add the travis build badge to README.md
How should this be manually tested?
Check the README.md file on ch-add-continuous-integration-163028251 branch to see the badge
### Screenshots
![screenshot from 2019-01-08 01-24-08](https://user-images.githubusercontent.com/30015297/50797058-8da8bc80-12e4-11e9-9ec8-d717dd8b1909.png)

### Any relevant pivotal tracker stories
[#163028251](https://www.pivotaltracker.com/story/show/163028251)
